database renaming, made models generic, migrations, changed code for new models, and deleted now unnecessary import commands

daveeloo opened the pull request.

paulproteus commented:
This looks great. I am too sleepy apparently to review immediately but intend to tomorrow.

paulproteus commented:
By "looks great" I mean that very tentatively, not in the sense of "looks good to me" that often accompanies merging things.

paulproteus commented:
On the whole, my feedback:
- There's a few cases where code quality could be improved (e.g. clarifying magic numbers). For those, file bugs and/or fix in follow-up commits.
- The file deletion is not adequately justified, so I can't merge this pull request without more clarity on that stuff. (Next step: Either create a separate file deletion pull request now, or just use "rebase -i" to drop that commit from this pull request.)
- The "lastest" word (typo?) you should fix. I'd be OK with seeing that in a follow-up patch, if you don't feel like redoing the pull request entirely.
- You should actually reply to every comment in the pull request.

So, if you feel like re-doing the pull request, there's enough stuff here to make that a reasonable option. But if you want to just ditch the file removal part, and then add follow-up commits to the rest, that'd be fine too. Be sure to leave a reply in the comments with how you plan to address each comment.

daveeloo committed:
Removed db_tables where not needed

daveeloo committed:
Created migration to change database names from uploads to greenhouse

daveeloo committed:
Changed models to be more generic
-Made changes to models.py
-Performed schema and data migrations

daveeloo committed:
Changed code to deal with migrated database
-Changed views to use new models
-Changed templates to use new view
-Changed import files to use new models

daveeloo committed:
Made whitespace changes

paulproteus commented:
I'm going to merge this unless you stop me!
